### PR TITLE
Add month range pickers to graphs

### PR DESCRIPTION
**Pull request by Afani97**
- Update `AgencyViewSet` to return grouped results by month and week, as well as year as today. 
- Add `MonthRangePicker` component 
- Update `TrafficStops` component to use `MonthRangePicker` in the traffic stops by count graph. 

TODO:
- [ ] Fix tests

### CODE REVIEW

File: nc/models.py [hunk 1]
```diff
@@ -203,7 +203,7 @@ class StopSummary(pg.ReadOnlyMaterializedView):
 
     id = models.PositiveIntegerField(primary_key=True)
     year = models.IntegerField()
```
Comment (Afani97) [inline nc/models.py:205]: I might remove `year` to 1) reduce the extra records created by the `date` and 2) I can just format the date to year like it's done for month and week. @copelco Any thoughts?

Comment (copelco) [reply nc/models.py:205]: I opened #189 as a suggested improvement for this

**Review by copelco — COMMENTED** [review]
Added one comment. I think this PR needs to be updated with the latest from `develop`.

**Comment by Afani97** [comment]
@copelco While I fix the backend tests, when you have a minute can you test this, specifically the Traffic Stops by Count and Use of Force graphs for now.

**Comment by copelco** [comment]
The graph doesn't filter down to year ranges for me -- do you see similar results? I rebuilt the PostgreSQL view, but maybe I need to do something else first?

https://user-images.githubusercontent.com/129679/234321550-daed30c0-73b6-46d0-b3e3-70284c9ffc9e.mov

**Comment by Afani97** [comment]
@copelco With the latest bug fixes, that should solve the graphs not updating for you.

**Review by copelco — APPROVED** [review]

**Pull request merged**